src/engine/QueryExecutionTree.h

```diff
@@ -200,9 +205,8 @@ class QueryExecutionTree {
       const vector<std::optional<pair<size_t, ResultTable::ResultType>>>&
           validIndices) const;
 
-  void writeTable(
+  ad_utility::stream_generator::stream_generator writeTable(
```

Comment:
Can this method be static?

Reply:
It can't getResult() is a non-static call

Reply:
It can't getResult() is a non-static call